pkg/buildah/version/version.go

```diff
@@ -0,0 +1,21 @@
+package version
+
+import (
+	"os/exec"
+
+	"github.com/kanopy-platform/buildah-plugin/pkg/buildah/common"
+)
+
+const (
+	Command = "version"
+)
+
+type (
+	CommandArgs struct{}
+)
+
+func (c *CommandArgs) GetCmds() []*exec.Cmd {
+	return []*exec.Cmd{
+		exec.Command(common.BuildahCmd, Command),
```

Comment:
`buildah version` command is mostly here as a sanity check and to show it works.